api/src/main/java/org/openmrs/module/idgen/service/db/IdentifierSourceDAO.java

```diff
@@ -142,6 +142,12 @@
      * @return the IdentifierSource with the given uuid
      */
     IdentifierSource getIdentifierSourceByUuid(String uuid);
+    
+    /**
+     * @param patientIdentifierType

```

Comment:
You could just use the @see annotation here

Reply:
removed the comments and put: `@see IdentifierSourceService#getIdentifierSourcesByType(PatientIdentifierType)`